functest/debug_test.go

```diff
@@ -55,7 +54,7 @@ func TestDebugSimple(t *testing.T) {
 		runChannel := make(chan error)
 		containerName := "testDebugSimpleContainer" + strings.ReplaceAll(stackRaw[i], "/", "_")
 		go func() {
-			_, err := cmdtest.RunAppsody(sandbox, "debug", "--name", containerName, "-P")
+			_, err := cmdtest.RunAppsody(sandbox, "debug", "--name", containerName)
```

Comment:
Should we do this change in this PR? Or did you just miss add it to the commit?

Reply:
Second answer. It's from where my master was ahead of appsody/master!